nncf/experimental/torch/fx/transformations.py

```diff
@@ -136,17 +142,47 @@ def bias_update_transformation(model: torch.fx.GraphModule):
     return bias_update_transformation
 
 
-def constant_update_transformation_builder(node: NNCFNode, value: torch.Tensor) -> TransformationFNType:
+def shared_constant_create_transformation(model: torch.fx.GraphModule):
+    """
+    Return transformation which checks fx graph for shared constants, disconnects
```

Comment:
This is a transformation, not a transformation builder, please correct the function name ( perhaps `shared_constants_unification_transformation`?)  and the description. Don't forget to mention the model param

Reply:
Done!